public/app/features/explore/LogRow.tsx

```diff
@@ -89,6 +90,10 @@ export class LogRow extends PureComponent<Props, State> {
   };
 
   onMouseOverMessage = () => {
+    const { parsingEnabled } = this.props;
+    if (!parsingEnabled) {
+      return;
```

Comment:
Have you tried exiting early here if `window.getSelection().anchorNode` is not null? If that worked, you could remove the mouse logic.

Reply:
Interesting. I have not. Will try it out :+1:

Reply:
@davkal Your suggestion doesn't feel as reliable, see screenshot below:
* Start with clicking on the page so nothing has focus
* Then I hover a row and I get null as anchorNode (row 1)
* Then I select text and I get multiple non-null as anchorNode (row 2 to 8)
* Then I click on a row to remove the text selection
* Then I hover log messages and I still get non-null as anchorNode (row 9 to 10)

![image](https://user-images.githubusercontent.com/1668778/57921794-8602b100-789e-11e9-9914-761f7b0681be.png)

Maybe looking at the [isCollapsed](https://developer.mozilla.org/en-US/docs/Web/API/Selection/isCollapsed) would be a viable solution, but not sure since the [selection object](https://developer.mozilla.org/en-US/docs/Web/API/Selection) is marked as experiemental technology.

Reply:
Seems like using the `isCollapsed` works at least in chrome and firefox. Have pushed changes which uses selection api.